src/lib.rs

```diff
@@ -50,15 +50,16 @@
 //! let event_loop = EventLoop::new().unwrap();
 //! let window = WindowBuilder::new().build(&event_loop).unwrap();
 //!
+//! // ControlFlow::Poll continuously runs the event loop, even if the OS hasn't
+//! // dispatched any events. This is ideal for games and similar applications.
+//! event_loop.set_control_flow(ControlFlow::Poll);
+//!
+//! // ControlFlow::Wait pauses the event loop if no events are available to process.
+//! // This is ideal for non-game applications that only update in response to user
+//! // input, and uses significantly less power/CPU time than ControlFlow::Poll.
+//! event_loop.set_control_flow(ControlFlow::Wait);
+//!
```

Comment:
This section is a bit weird, it's like we have a code, which sets control flows and explains what they do, but it just one discards the other?

Reply:
This documentation has to be improved anyway. But I didn't add it in this PR, I just moved it to reflect the fact that we can change this now *before* `run()`.

We should tackle this in a different PR.

Reply:
I'd just simply drop it entirely, but could be done in a separate PR.